SB/index.js

```diff
@@ -23,6 +23,7 @@ app.post('/addPerson', (req, res) => {
 
 //DELETE Route (/deletePerson)
 app.delete('/deletePerson', (req, res) => {
+    const { id } = req.params;
```

Comment:
on line 26, when initializing the 'id' , all you need is to add '.id' to the end of req.params because that is how you identify the key you need from the query in the URL. Otherwise, the current 'id'  may get initialized with unintended values.

Reply:
you could also add ' .["id"] ' instead of '.id' . It depends on your preference.

Reply:
fixed, Thanks for your help